exonum-java-binding/core/rust/Cargo.toml

```diff
@@ -29,6 +28,13 @@ serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0.40"
 
+[dependencies.exonum]
+# FIXME: using git dependency until new Exonum with MerkleDB will be released
```

Comment:
This is a good idea, but I'm afraid we cannot deal with that version error without the `[patch.crates-io]` section. I'd suggest moving underlying lines there.